framework/src/main/java/org/checkerframework/framework/type/GenericAnnotatedTypeFactory.java

```diff
@@ -1702,6 +1702,20 @@ public AnnotatedTypeMirror getAnnotatedTypeLhs(Tree lhsTree) {
         return res;
     }
 
+    /**
+     * As long as everUseFlow is true, always enable flow refinement for the receiver.
+     *
+     * @param tree an expression tree
+     * @return the refined type of the expression tree
+     */
+    public AnnotatedTypeMirror getAnnotatedTypeWithReceiverRefinement(Tree tree) {
```

Comment:
change to `package-private`

Reply:
improve the document, what is for, nobody outside should use it. @ see to other method which would usually be used.